specs/core/shapes/Comment-shape.ttl

```diff
@@ -16,8 +16,8 @@
 @prefix oslc:  <http://open-services.net/ns/core#> .
 @prefix rdf:   <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
 @prefix dcterms: <http://purl.org/dc/terms/> .
-
-@base <http://open-services.net/ns/core/shapes/3.0/Comment-shape.ttl> .
+@prefix oslc_s: <https://open-services.net/ns/core/3.0/shapes/ps01/#> .
```

Comment:
where is the prefix used? If its needed, can you include it in the base so that the content isn't duplicated?

Reply:
all these files will be deleted